demo-site/scripts/create_static_files.js

```diff
@@ -0,0 +1,105 @@
+//region imports
```

Comment:
I think this is also not needed - it was originally meant to create static files based on the provided template for ALL servers, filling in the app_id, enforcer_name, sensor_src, etc. as needed. Here, the only thing that might change is sensor src (first party or third party). Everything else is static.

Reply:
Removed from dockerFile + deleted the file.